Tests/ParseSwiftTests/ParseObjectBatchTests.swift

```diff
@@ -665,7 +665,7 @@ class ParseObjectBatchTests: XCTestCase { // swiftlint:disable:this type_body_le
             }
 
         }
-        wait(for: [expectation1, expectation2], timeout: 10.0)
+        wait(for: [expectation1, expectation2], timeout: 30.0)
```

Comment:
Same here; is there a reason this timeout is going up?

Reply:
This gives more time if it's needed on slower machines. If it's fulfilled earlier it won't hit this timeout